packages/utils/src/SDKErrors.ts

```diff
@@ -174,9 +174,21 @@ export class IdentityMismatchError extends SDKError {
   }
 }
 
-export class WsAddressNotSetError extends SDKError {
-  constructor() {
-    super('Node address to connect to not configured!')
+export class FullNodeConnectionNotSetError extends SDKError {
+  constructor(options?: ErrorOptions) {
+    super(
+      'A function tried to access the ApiPromise instance before it was set. Did you forget to call `Kilt.init({ api })`?',
+      options
+    )
+  }
+}
+
+export class SubscriptionsNotSupportedError extends SDKError {
+  constructor(options?: ErrorOptions) {
+    super(
+      'The api instance was used in a context that relies on state or event subscriptions, but the provider does not support them',
```

Comment:
I’d like to have a shorter message and to point the developer in the right direction here.
```suggestion
      'The blockchain API should support state or event subscriptions, for example `WsProvider`.',
```